### ISSUE TITLE
Confusing 2FA setup

### ISSUE THREAD

**Issue by 0dinD** [HangarMC/Hangar#1165]
### Observed/problematic behavior

I was trying to set up 2FA, but it took me a while until I understood what Hangar wanted me to do, the UX was a bit confusing for me.

### Expected behavior

I expected the UI to guide me through the steps required to set up 2FA.

### Steps to reproduce

First, I was on this page, and scanned the QR code using Google Authenticator:

![settings-authenticator-app](https://user-images.githubusercontent.com/52179873/233813174-74eac8c1-98b6-451c-951f-ee179babad10.png)

After scanning the code, it wan't immediately obvious what the next step was (there is no guiding text), but I assumed I had to click the "Verify code and activate" button. In hindsight, I now understand that I was also supposed to enter a TOTP code from my authenticator app into the "Code" text field, but I initially missed that part since there was no guiding text, and Hangar did not immediately complain when I left the text field empty. I also initially thought that "Code" was referring to the text above it, "Can't scan? enter the code listed below the image!", but now I realize that this refers to a different "code". After clicking the button (and "accidentally" leaving the "Code" field empty), I got this dialog:

![confirm-backup-codes](https://user-images.githubusercontent.com/52179873/233813177-f8a8c9eb-d756-46c3-8c48-6ac9643895b6.png)
![confirm-saved](https://user-images.githubusercontent.com/52179873/233813182-459086fa-42a3-4d98-8e96-a30ef2d21338.png)

Here, I entered one of the backup codes, but got an error that just said "Code is required". This was confusing to me, because I thought it was referring to the backup codes, but it was probably referring to the "Code" field, which I couldn't even see since it was behind the dialog. I then tried some variations of the code, and also got a different error that said "You need to be in a priviliged session to do this", which was even more confusing to me.

Eventually I managed to figure it out, but I thought the whole process was a bit confusing :slightly_smiling_face: 

### Other

To summarize, I think there are a few different problems here:

1. The "Code" text field and "Verify and activate" button has no descriptive/helpful text above it to explain what/why you're supposed to do. There is text immediately above it, which explains how to scan the QR code, but this has nothing to do with the "Code" field, which is confusing.
2. The "Verify code and activate" button should probably require/validate the "Code" field before opening the dialog, so you can't accidentally leave it empty and get confused.
3. There are multiple different, ambiguous "codes" being referred to in the texts/fields/buttons.
    - The code below the QR code
    - The TOTP code
    - The backup code(s)
